students/minchan/westagram/users/views.py

```diff
@@ -1,3 +1,47 @@
-from django.shortcuts import render
+import json
+import re
+from django.http.response   import JsonResponse
+from django.views           import View
+from users.models           import Users
+from django.http            import HttpResponse
 
 # Create your views here.
+class UserRegister(View) :
+    REGEX_EMAIL     = re.compile("[@][a-zA-Z]*[.]")
+    REGEX_PASSWORD  = re.compile("/^(?=.*[a-zA-Z])(?=.*\d)(?=.*[`~!@#$%^&*(),<.>/?]).{8,}")
+
+    def valid_email(self,email) :
+        return True if self.REGEX_EMAIL.match(email) else False
+
+    def is_email_duplicate(self,email) :
+        return Users.objects.filter(email=email).exists()
+    
+    def valid_pw(self,pw) :
+        return True if self.REGEX_PASSWORD.match(pw) else False
+
+    def post(self,request) :
+        data    = json.loads(request.body)
+        email   = data["email"]
+        pw      = data["password"]
```

Comment:
아직 수정이 필요한 부분이 남아 있네요!

```suggestion
        password      = data["password"]
```

Reply:
password로 변경하고, 나머지 변수와 줄맞춤 완료했습니다.

Reply:
민찬님 `file change`에서 확인해보면 반영이 안되어 있네요!?
commit 내역 한번 확인해 주실 수 있나요?

Reply:
이게 저장을 안누르고 커밋을해서 그런건지
버전이 한번 씩 밀린 상태로 커밋이 되고 있었네요.
한번 더 커밋/푸시 완료했습니다.